src/widgets/hero/ui/hero.tsx

```diff
@@ -6,8 +8,8 @@ export const Hero = () => {
       <div className="main content">
         <div className="title-container">
           <div className="subtitle-container">
-            <h2 className="subtitle">an international community of developers</h2>
-            <h2 className="subtitle">since 2013</h2>
+            <Subtitle className="subtitle" color="black">an international community of developers</Subtitle>
```

Comment:
Why are you adding the "subtitle" class twice? I don't think you need it.

![image](https://github.com/user-attachments/assets/393c8483-7a7f-4506-8be1-7bf7fff41d53)